registry.yaml

```diff
@@ -3077,7 +3080,7 @@ packages:
     amd64: x86_64
     arm64: aarch64
     darwin: apple-darwin
-    linux: unknown-linux-musl
+    linux: unknown-linux-gnu
```

Comment:
Why is this change needed?
In my understanding, `musl` is more portable than `gnu` because `musl` is built by static link.
I'm not familiar with this field, so please let us know the reason.

Reply:
Because, only `unknown-linux-gnu` binary is provided for `aarch64`.
If we can change `OS` value when architecture is arm64, it is better.

ref: https://github.com/sharkdp/bat/releases/tag/v0.19.0

Reply:
Oh, I see. Thank you for your explanation.

Reply:
https://github.com/aquaproj/aqua-registry/pull/2153/commits/ac2c81e66d0c5f59da0c47a6d33ce9ea45e8b7e1

Reply:
Thank you.